html/semantics/forms/attributes-common-to-form-controls/resources/dirname.js

```diff
@@ -0,0 +1,20 @@
+function __onIframeLoaded(t, cb, selector, done) {
+  const iframe = document.querySelector(selector);
+  iframe.addEventListener("load", function() {
+    // The initial about:blank load event can be fired before the form navigation occurs.
+    // See https://github.com/whatwg/html/issues/490 for more information.
+    if(iframe.contentWindow.location.href == "about:blank") { return; }
+
+    const params = new URLSearchParams(iframe.contentWindow.location.search);
+    t.step(() => cb(params))
+    if (done) t.done();
+  });
+}
+
+function onIframeLoaded(t, cb, selector="iframe") {
+  return __onIframeLoaded(t, cb, selector, false);
+}
```

Comment:
Am I blind or are there no callers for this algorithm?

Reply:
Yeah, I wasn't sure if I needed this later on, so I included both versions. Do you want me to remove it?